### PR TITLE
html 28 review code styled

### PR DESCRIPTION
**Pull request by cajimenez96**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    background: var(--test);
+    border: 2px solid var(--btn-border-2);
+    transition: .3s all ease;
+}
+.btn-accept:hover{
+    color: var(--font-color-2);
+    background: var(--btn-secondary);
+    border: 2px solid var(--btn-border);
+    transition: .3s all ease;
+}
```
Comment (patoogm) [inline src/App.css:33]: Please, enter the end of line

File: src/components/Footer/footer.css [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+  font-size: 14px;
+}
+.box-footer .box-info p{
+  font-size: 12px;
+}
+.box-social-icons a i{
+  margin: 0 10px;
+  font-size: 24px;
+  color: var(--font-color);
+}
```
Comment (patoogm) [inline src/components/Footer/footer.css:15]: Please, enter the end of line

File: src/components/Hero/hero.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    }
+    .hero-container .box-info-hero{
+        margin-top: 20px;
+    }
+    .hero-container .box-suggestion h5{
+        margin: 20px 10px;
+        margin-bottom: 10px;
+        padding-left: 10px;
+    }
+}
```
Comment (patoogm) [inline src/components/Hero/hero.css:23]: Please, enter the end of line

File: src/components/ManageSinister/ManageSinister.js [hunk 4]
```diff
@@ -2,111 +2,129 @@ import React from 'react';
...
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="domicilio-tercero" className="form-label">Domicilio</label>
+          <input type="text" className="form-control" id="domicilio-tercero" required />
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="telefono-tercero" className="form-label">Telefono de contacto</label>
+          <input type="text" className="form-control" id="telefono-tercero" required />
```
Comment (patoogm) [inline src/components/ManageSinister/ManageSinister.js:60]: type=text is correct?

File: src/components/ManageSinister/ManageSinister.js [hunk 5]
```diff
@@ -2,111 +2,129 @@ import React from 'react';
...
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-5" data-aos="fade-up">
+          <label htmlFor="apellido-tercero" className="form-label">Apellido</label>
+          <input type="text" className="form-control" id="apellido-tercero" required />
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="documento-tercero" className="form-label">Documento</label>
+          <input type="text" className="form-control" id="documento-tercero" required />
```
Comment (patoogm) [inline src/components/ManageSinister/ManageSinister.js:50]: type=text is correct?

File: src/components/ManageSinister/ManageSinister.js [hunk 6]
```diff
@@ -2,111 +2,129 @@ import React from 'react';
...
+          <label htmlFor="numero-siniestro" className="form-label">Número de siniestro</label>
+          <input type="text" className="form-control" id="numero-siniestro" required />
+          <div className="valid-feedback">Completo!</div>
         </div>
-        <div className='col-md-3'>
-          <label htmlFor='numero-denuncia' className='form-label'>Número de denuncia</label>
-          <input type='text' name='denunciaTer' className='form-control' id='numero-denuncia' required />
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="numero-denuncia" className="form-label">Número de denuncia</label>
+          <input type="text" className="form-control" id="numero-denuncia" required />
```
Comment (patoogm) [inline src/components/ManageSinister/ManageSinister.js:34]: type=text is correct?

File: src/components/ManageSinister/ManageSinister.js [hunk 7]
```diff
@@ -2,111 +2,129 @@ import React from 'react';
...
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-5" data-aos="fade-up">
+          <label htmlFor="apellido-solicitante" className="form-label">Apellido</label>
+          <input type="text" className="form-control" id="apellido-solicitante" required />
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="documento-solicitante" className="form-label">Documento</label>
+          <input type="text" className="form-control" id="documento-solicitante" required />
```
Comment (patoogm) [inline src/components/ManageSinister/ManageSinister.js:89]: type=text is correct?

File: src/components/ManageSinister/ManageSinister.js [hunk 8]
```diff
@@ -2,111 +2,129 @@ import React from 'react';
...
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="domicilio-solicitante" className="form-label">Domicilio</label>
+          <input type="text" className="form-control" id="domicilio-solicitante" required />
+          <div className="valid-feedback">Completo!</div>
+        </div>
+        <div className="col-md-3" data-aos="fade-up">
+          <label htmlFor="telefono-solicitante" className="form-label">Telefono de contacto</label>
+          <input type="text" className="form-control" id="telefono-solicitante" required />
```
Comment (patoogm) [inline src/components/ManageSinister/ManageSinister.js:99]: type=text is correct?

File: src/components/Navbar/navbar.css [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+.navbar-styles .logo-box .link-img img{
+    width: 220px;
+    padding: 5px;
+    transition: all 0.3ms ease;
+    border: none;
+}
+.navbar-styles .navbar-text span{
+    color: var(--font-color);
+    font-size: 18px;
+}
```
Comment (patoogm) [inline src/components/Navbar/navbar.css:18]: Please, enter the end of line

**Review by patoogm — CHANGES_REQUESTED** [review]

**Comment by Nidev-lab** [comment]
Please, resolve this problem too..
image

File: src/components/Footer/Footer.js [hunk 10]
```diff
@@ -4,34 +4,33 @@ import logo from '../../assets/img/logo.png';
...
-                <i className='bi bi-instagram fs-1 text-black'></i>
-              </a>
-            </div>
+        <div className="d-sm-flex justify-content-center flex-column mt-3">
+          <span className="text-center fw-bold">Redes Sociales</span>
+          <div className="d-sm-flex justify-content-center m-sm-3 text-center box-social-icons">
+            <a href="">
+              <i className="bi bi-facebook"></i>
+            </a>
+            <a href="">
```
Comment (patoogm) [inline src/components/Footer/Footer.js:22]: Please, correct. href="/#"

**Review by patoogm — APPROVED** [review]

**Pull request merged**